app/code/Magento/ConfigurableProduct/Test/Unit/Plugin/Model/ResourceModel/ProductTest.php

```diff
@@ -31,18 +39,45 @@ class ProductTest extends \PHPUnit\Framework\TestCase
      * @var \Magento\ConfigurableProduct\Plugin\Model\ResourceModel\Product
      */
     private $model;
+    /**
+     * @var ProductAttributeRepositoryInterface|\PHPUnit\Framework\MockObject\MockObject
+     */
+    private $prdAttributeRepositoryMock;
+    /**
+     * @var SearchCriteriaBuilder|\PHPUnit\Framework\MockObject\MockObject
+     */
+    private $searchCriteriaBuilderMock;
+    /**
+     * @var FilterBuilder|\PHPUnit\Framework\MockObject\MockObject
+     */
+    private $filterBuilderMock;
 
     public function setUp()
     {
         $this->objectManager = new \Magento\Framework\TestFramework\Unit\Helper\ObjectManager($this);
         $this->configurableMock = $this->createMock(Configurable::class);
         $this->actionMock = $this->createMock(ActionInterface::class);
+        $this->prdAttributeRepositoryMock = $this->getMockBuilder(ProductAttributeRepositoryInterface::class)
```

Comment:
Please rename the property accordingly
```suggestion
        $this->productAttributeRepositoryMock = $this->getMockBuilder(ProductAttributeRepositoryInterface::class)
```